### PR TITLE
Use Arcade global tool versioning

### PR DESCRIPTION
**Pull request by tmat**
The next package version will be `3.1.xxx`, where `xxx` is auto-generated number.

Also cleans up resx generation.

### CODE REVIEW

**Comment by tmat** [comment]
@JoeRobich

File: src/dotnet-format.csproj [hunk 1]
```diff
@@ -3,19 +3,23 @@
...
     <OutputType>Exe</OutputType>
-    <LangVersion>7.3</LangVersion> 
+    <LangVersion>7.3</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <RootNamespace>Microsoft.CodeAnalysis.Tools</RootNamespace>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+    <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
+
     <Description>Command line tool for formatting C# and Visual Basic code files based on .editorconfig settings.</Description>
-    <PackageIconUrl>https://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
```
Comment (jmarolf) [inline src/dotnet-format.csproj]: How is the icon generated? Does aracde do this automatically?

Comment (tmat) [reply src/dotnet-format.csproj]: No change with icon, just moved the line.

File: src/dotnet-format.csproj [hunk 2]
```diff
@@ -3,19 +3,23 @@
...
 
-    <ServerGarbageCollection>true</ServerGarbageCollection>
     <IsPackable>true</IsPackable>
     <PackAsTool>true</PackAsTool>
-    <!-- These identifiers are for generating the shim'd core executables for signing. Not all options
-         from $(RoslynPortableRuntimeIdentifiers) work or make sense in this context. -->
+    <PackageIconUrl>https://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
```
Comment (jmarolf) [inline src/dotnet-format.csproj]: Oh I see, sorry :)

**Pull request merged**